Library/Homebrew/cmd/test-bot.rb

```diff
@@ -155,6 +156,14 @@ def run
           working_dir.cd { exec(*@command) }
         end
         write.close
+        if travis
+          print_thread = Thread.new do
+            while true do
+              printf "."
+              sleep 60
+            end
+          end
+        end
```

Comment:
I don't think we should do this for every test. We only need it for `brew install`.